join-statements/joinstatements.md

```diff
@@ -4,63 +4,33 @@
 
 **CROSS JOIN** - it shows all possible combinations between rows from table A and rows from table B, so a result from the query would be <# of A rows> x <# of B rows>
 
-A real world example would be if I had a table with fruit juices, and another table with entrees at a high school cafeteria. A CROSS JOIN would allow me to compare all the possible combinations of meals for the high schoolers.
-
-```
-SELECT m.dish, d.drink
-FROM meals AS m
-CROSS JOIN drink AS d;
-```
+A real world example would be if a nutritionist wanted to evaluate all the possible meal combinations that kids in highschool could have at lunch. If each kid were to choose a drink, and a main serving the nutritionist would want to verify that the highschool is allowing enough variety on a day to day basis. `CROSS JOIN` would allow me to compare all the possible combinations of meals for the high schoolers and allow the nutritionists to evaluate the combinations.
 
 **INNER JOIN** - it's the result of a query between 2 tables where the output is the intersection of table A and table B that satisfies a specified conditional statement between both tables using the `ON` statement.
 
-A real world example would be if, there were a table of students being specified in the `FROM` portion of the query, and another table of different extracurricular societies that keeps records of students by their student ID. We could use the `INNER JOIN`, to get a list of students that are found in a given society using an INNER JOIN between these two tables. The `INNER JOIN` would return matches as specified in the `ON` statement. The query would look something like this.
+A real world example would be of the head guidance counselor looking to assign students to his peers. He would like to create a list of students and the society they are in, in order for him to best match each student to a given guidance counselor based on the society they are in.
```

Comment:
how does society relate to guidance counselor?

Reply:
done